### PR TITLE
[DI] Add SDK versions and API versions relationship in README.md

### PR DESCRIPTION
**Pull request by YalinLi0312**

### CODE REVIEW

File: sdk/documentintelligence/azure-ai-documentintelligence/README.md [hunk 1]
```diff
@@ -19,6 +22,13 @@ Azure AI Document Intelligence ([previously known as Form Recognizer][service-re
 python -m pip install azure-ai-documentintelligence
 ```
 
+This table shows the relationship between SDK versions and supported API versions of the service:
+|SDK version|Supported API version of service
+|-|-
+|1.0.0b1 | 2023-10-31-preview
+
+Older API versions are supported in `azure-ai-formrecognizer`, please check the relationship from [there](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/README.md#install-the-package).
```
Comment (catalinaperalta) [inline sdk/documentintelligence/azure-ai-documentintelligence/README.md:30]: Should we focus on linking the migration guide to encourage folks to move on to this new package?

File: sdk/documentintelligence/azure-ai-documentintelligence/README.md [hunk 2]
```diff
@@ -19,6 +22,13 @@ Azure AI Document Intelligence ([previously known as Form Recognizer][service-re
 python -m pip install azure-ai-documentintelligence
 ```
 
+This table shows the relationship between SDK versions and supported API versions of the service:
+|SDK version|Supported API version of service
```
Comment (catalinaperalta) [inline sdk/documentintelligence/azure-ai-documentintelligence/README.md:26]: Nit: 
```suggestion
|SDK version|Supported service API version
```

File: sdk/documentintelligence/azure-ai-documentintelligence/README.md [hunk 3]
```diff
@@ -585,19 +595,17 @@ additional questions or comments.
...
 [python-di-pypi]: https://pypi.org/project/azure-ai-formrecognizer/
-[python-di-product-docs]: https://learn.microsoft.com/azure/applied-ai-services/form-recognizer/overview?view=form-recog-3.0.0
-[python-di-ref-docs]: https://aka.ms/azsdk/python/formrecognizer/docs
 [python-di-samples]: https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/documentintelligence/azure-ai-documentintelligence/samples
 
 [azure_portal]: https://ms.portal.azure.com/
 [regional_endpoints]: https://azure.microsoft.com/global-infrastructure/services/?products=form-recognizer
 [FR_or_CS_resource]: https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account?tabs=multiservice%2Cwindows
 [cognitive_resource_portal]: https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesFormRecognizer
+https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesDocumentIntelligence
```
Comment (catalinaperalta) [inline sdk/documentintelligence/azure-ai-documentintelligence/README.md:608]:
```suggestion
[cognitive_resource_portal]: 
https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesDocumentIntelligence
```

**Review by catalinaperalta — APPROVED** [review]
FYI there is a link that needs to be fixed before merging

File: sdk/documentintelligence/azure-ai-documentintelligence/README.md (hunk 3 above)
Comment (YalinLi0312) [reply sdk/documentintelligence/azure-ai-documentintelligence/README.md:608]: No, the portal link haven't changed.

**Comment by YalinLi0312** [comment]
> FYI there is a link that needs to be fixed before merging

The portal link is same, it's still ends with "FormRecognizer". The one you proposed doesn't exist:

**Review by mssfang — APPROVED** [review]

File: sdk/documentintelligence/azure-ai-documentintelligence/README.md (hunk 1 above)
Comment (YalinLi0312) [reply sdk/documentintelligence/azure-ai-documentintelligence/README.md]: Added migration guide link.

**Pull request merged**